vars/fcosKola.groovy

```diff
@@ -21,7 +21,7 @@ def call(params = [:]) {
             // Add the tests/kola directory, but only if it's not the same as the
             // src/config repo which is also automatically added.
             if (shwrapRc("""
-                test -d tests/kola
+                test -d ${cosaDir}/tests/kola
```

Comment:
`cosaDir` is where we do cosa builds, but not necessarily where the project's git repo is checked out, right? This code is assuming that we're in `WORKSPACE`, so maybe what we really need is

```suggestion
                test -d ${env.WORKSPACE}/tests/kola
```

?

Reply:
Oh yes...done.